libs/bluevk/CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ target_link_libraries(${TARGET} utils math)
 target_include_directories(${TARGET} PUBLIC ${PUBLIC_HDR_DIR})
 
 # test_bluevk is not supported on mobile or Windows
-if (NOT ANDROID AND NOT IOS AND NOT WIN32)
+if (NOT ANDROID AND NOT IOS AND NOT WIN32 AND NOT FILAMENT_SKIP_SDL2)
```

Comment:
Don't you need to also skip libapp and the native samples when FILAMENT_SKIP_SDL2 is set?

Reply:
samples only build in case of host

Reply:
The two use cases are tools and target -dev package build.

```
EXTRA_OECMAKE:class-native += " \
    -D BUILD_SHARED_LIBS=OFF \
    -D CMAKE_BUILD_TYPE=Release \
    -D FILAMENT_SUPPORTS_VULKAN=ON \
    -D FILAMENT_SUPPORTS_OPENGL=OFF \
    -D FILAMENT_BUILD_FILAMAT=OFF \
    -D FILAMENT_SKIP_SAMPLES=ON \
    -D FILAMENT_SKIP_SDL2=ON \
    ${PACKAGECONFIG_CONFARGS} \
    "

EXTRA_OECMAKE:class-target += " \
    -D BUILD_SHARED_LIBS=OFF \
    -D CMAKE_BUILD_TYPE=Release \
    -D FILAMENT_SUPPORTS_VULKAN=ON \
    -D FILAMENT_SUPPORTS_OPENGL=OFF \
    -D FILAMENT_LINUX_IS_MOBILE=ON \
    -D IMPORT_EXECUTABLES_DIR=. \
    -D FILAMENT_SKIP_SDL2=ON \
    -D DIST_ARCH=${BUILD_ARCH} \
    -D FILAMENT_HOST_TOOLS_ROOT=${STAGING_BINDIR_NATIVE} \
    ${PACKAGECONFIG_CONFARGS} \
    "
```